cypress-io/cypress
# feat: scan for 3rd party ct plugins

WIP

Example repo with [cypress-ct-solidjs](https://github.com/ZachJW34/cypress-ct-detect-3rd-party)

- Closes #25680

### Additional details


### Steps to test
Clone the [cypress-ct-solidjs](https://github.com/ZachJW34/cypress-ct-detect-3rd-party)
Run `yarn && yarn cypress:open` within cypress and point it at the cloned repo. The dep should be detected when setting up CT Testing.

### How has the user experience changed?

https://user-images.githubusercontent.com/25158820/217650396-26d91d6b-1240-43f1-a786-3aa487897eda.mov



### PR Tasks
